docs/fleet/configure-a-fleet.md

```diff
@@ -78,7 +78,262 @@ For an example of adding a fragment to a machine, see the [Add a Rover Fragment
+   - Sets the pin number for pin `a` to `30`.
+   - Sets the name of the board associated with this motor to `local`.
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   "fragment_mods": [
+    {
+      "fragment_id": "abcd7ef8-fa88-1234-b9a1-123z987e55aa",
+      "mods": [
+        {
+          "$set": {
+            "components.motor1.attributes.max_rpm": 1818
+          }
+        },
+        {
+          "$set": {
+            "components.motor1.name": "my_motor"
+          }
+        },
+        {
+          "$set": {
+            "components.motor1.attributes.pins.a": 30
+          }
+        },
+        {
+          "$set": {
+            "components.motor1.attributes.board": "local"
+          }
+        }
+      ]
+    }
+   ],
+   ```
+
+   {{% /expand%}}
+   {{%expand "Remove an attribute" %}}
+   This example uses [`$unset`](https://www.mongodb.com/docs/manual/reference/operator/update/unset/#mongodb-update-up.-unset) to remove whatever pin number was set for the `pwm` pin, so the motor no longer has a PWM pin set.
+   In other words, it deletes the `pwm` pin field.
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   "fragment_mods": [
+     {
+       "fragment_id": "abcd7ef8-fa88-1234-b9a1-123z987e55aa",
+       "mods": [
+         {
+          "$unset": {
+            "components.motor1.attributes.pins.pwm": 0
+          }
+         }
+       ]
+     }
+   ],
+   ```
+
+   {{% /expand%}}
+   {{%expand "Modify dependencies" %}}
+   This example uses [`$set`](https://www.mongodb.com/docs/manual/reference/operator/update/set/#mongodb-update-up.-set) to assign a new list of dependencies to a component named `rover_base2`.
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   "fragment_mods": [
+     {
+       "fragment_id": "abcd7ef8-fa88-1234-b9a1-123z987e55aa",
+       "mods": [
+         {
+          "$set": {
+            "components.rover_base2.attributes.depends_on": ["local", "motor1"]
+          }
+         }
+       ]
+     }
+   ],
+   ```
+
+   {{% /expand%}}
+   {{%expand "Change motor pins from A and B to PWM and DIR" %}}
+   This example uses [`$rename`](https://www.mongodb.com/docs/manual/reference/operator/update/rename/) to make the following changes to the attributes of a motor named `motor1` in the fragment:
+
+   - Takes whatever the pin number for pin `a` was and assigns that value to the PWM pin.
+     Deletes the `pins.a` field.
+   - Takes whatever the pin number for pin `b` was and assigns that value to the DIR pin.
+     Deletes the `pins.b` field.
+
+   _If you want to change the `name` of a component (for example, `motor1`), use `$set`, as shown in the change the name of a component example.
+   `$rename` is for changing the key, not the value._
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   "fragment_mods": [
+    {
+      "fragment_id": "abcd7ef8-fa88-1234-b9a1-123z987e55aa",
+      "mods": [
+        {
+          "$rename": {
```

Comment:
stacking would have to be here too. Sorry I missed this earlier.
Also forgot to mention that the docs look really nice - thank you 😊